### PR TITLE
Executive 2020 09 11

### PR DESCRIPTION
**Pull request by LongForWisdom**
Don't merge yet, still contains unconfirmed things.

### CODE REVIEW

File: governance/votes/Executive vote - September 11, 2020.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+...
+
+## Summary
+
+Therefore, if this Executive proposal passes the following will occur:
+- The USDC-A Debt Ceiling will be increased from 40 million to 100 million.
+- The Vat global `line` parameter will increase from 763 million to 823 million.
+- [tBTC](https://tbtc.network/) will be whitelisted on the ETHBTC Oracle.
+...
```
Comment (gbalabasquer) [inline governance/votes/Executive vote - September 11, 2020.md:29]: Remove the `...`?

**Review by gbalabasquer — DISMISSED** [review]
Just a minor comment but looks good to me.

File: governance/votes/Executive vote - September 11, 2020.md (hunk 1 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - September 11, 2020.md:29]: Yep, thanks.

**Comment by LongForWisdom** [comment]
Can merge now. 60M USDC-A raise is confirmed.

**Review by derek-flossman — APPROVED** [review]

**Comment by gbalabasquer** [comment]
Yes please go ahead. I don't have permissions to do it.

**Pull request merged**